### ISSUE TITLE
on Cygwin ECL does not handle case-sensitve paths well

### ISSUE THREAD

**Issue by sageb0t** [sageb0t/testsage#13343]
Imported from <a href="http://trac.sagemath.org/sage_trac/ticket/13343">trac #13343</a>.

ECLs call 

```
> (directory "")
```

must always return the current directory. On Cygwin, sometimes one gets NIL. One example is like this: say, a user John has Windows (domain) login name `John`, but Cygwin will create for him home directory "/home/john".
At this point, one might hit the [http://cygwin.com/cygwin-ug-net/using-specialnames.html#pathnames-casesensitive Win32 case-sensitivity weirdness], and in particular installation scripts for Maxima spkg will fail in a subdirectory containing "/home/john". If one switches on case-sensitivity, as described in the link just cited, the problem goes away.

In a nutshell, while this remains unfixed, avoid building Sage in directories with names containing capitals. This was uncovered and mentioned [https://groups.google.com/d/msg/sage-devel/Gj3cbdcPqKo/h5Ln_9qH0yYJ here].

Reported by: dimpase

Component: cygwin

CC: 

Report Upstream: Reported upstream. No feedback yet.

Authors: 

Dependencies: 

Work issues: 

Reviewers: 

Merged in: 

Attachments: <ol></ol>
